addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/launch-product/launch-product.js

```diff
@@ -365,26 +365,26 @@ class LaunchProduct extends StepBase {
   }
 
   /**
-   * Check if the resolved input parameter contains a static namespace param. If so, augments the namespace to begin with 
-   * 'analysis-' for permissions purposes (if it does not already start with that) and to end with a unique datetime string 
+   * Check if the resolved input parameter contains a static namespace param. If so, augments the namespace to begin with
+   * 'analysis-' for permissions purposes (if it does not already start with that) and to end with a unique datetime string
    * so Cloudformation doesn't create duplicate stacks for separate workspaces.
-   * 
-   * @param resolvedInputParams 
-   * @param datetime 
+   *
+   * @param resolvedInputParams
+   * @param datetime
    * @returns {namespaceParam:string, namespaceIndex:string}
    */
-  getNamespaceAndIndexIfNecessary(resolvedInputParams, datetime){
+  getNamespaceAndIndexIfNecessary(resolvedInputParams, datetime) {
     const namespaceIndex = resolvedInputParams.findIndex(element => element.Key === 'Namespace');
     let namespaceParam = resolvedInputParams[namespaceIndex].Value;
 
     // Check to make sure the resolved namespace variable begins with 'analysis-' so our templates will allow it
-    if(!namespaceParam.startsWith('analysis-')){
-      namespaceParam = 'analysis-' + namespaceParam;
+    if (!namespaceParam.startsWith('analysis-')) {
+      namespaceParam = `analysis-${namespaceParam}`;
     }
 
     // Check to make sure the resolved namespace variable ends with a unique datetime string so it will be unique for each deployment of a configuration with a static namespace
-    if(namespaceParam.split('-').pop() !== datetime.toString()){
-      namespaceParam += '-' + datetime;
+    if (namespaceParam.split('-').pop() !== datetime.toString()) {
```

Comment:
Since dateTime stamps are always unique, is this check even necessary?

Reply:
Yes, this will be false if the namespace was dynamic because the datetime parameter passed is the same one that was used to dynamically make the namespace above